### ISSUE TITLE
The plugin don't load

### ISSUE THREAD

**Issue by ghost** [jojoe77777/SlapperCooldown#1]
This plugin works? It might be useful for me but Igual try anyway to make it work but the plugin simply don't even load in the server

**Comment by Gianluxx** [comment]
show console loading if there are errors

**Comment by ghost** [comment]
Nop, I checked the console but the plugin don't appear, like if it doesn't exist, and it's the same API version

**Comment by Gianluxx** [comment]
Is it in the plugins folder?

**Comment by ghost** [comment]
Yep, it is, I though it was the API so I changed but still the same

**Comment by ghost** [comment]
I fix it, just use the Dev tools and I put it like a folder instead of phar file, it happens with other plugins and I have to do this, I don't know why🤔
